AnimateQuotes.js

```diff
@@ -29,51 +65,24 @@ export default function AnimateQuotes(props) {
         delay: QUOTES_IN_DELAY,
         useNativeDriver: true,
       }),
-    ]).start();
-    const endOfAnim = setTimeout(() => {
-      setIsRegistered(false);
-      clearTimeout(endOfAnim);
-    }, Math.max(SVG_OUT_DURATION, QUOTES_IN_DURATION) + QUOTES_IN_DELAY);
+    ]).start(() => {
+      setIsQuoteAnimationRunning(false);
+      setIsShakeReady(true);
+    });
+  };
+
+  // lands on Quotes after LogoSvg fades out first time
+  if (isShakeTriggered && !hasInitialTransitionRun && !isQuoteAnimationRunning) {
+    setIsQuoteAnimationRunning(true);
+    quoteAnimationIn();
     // Quotes fade out, delay for LogoSvg fade in/fade, then Quotes fade back in
-  } else if (toggle && hasToggledBefore && !isRegistered) {
-    setIsRegistered(true);
-    Animated.sequence([
-      Animated.parallel([
-        Animated.timing(fadeAnim, {
-          toValue: 0,
-          duration: QUOTES_OUT_DURATION,
-          useNativeDriver: true,
-        }),
-        Animated.timing(sizeAnim, {
-          toValue: 0.5,
-          duration: QUOTES_OUT_DURATION,
-          useNativeDriver: true,
-        }),
-      ]),
-      Animated.parallel([
-        Animated.timing(fadeAnim, {
-          toValue: 1,
-          duration: QUOTES_IN_DURATION,
-          delay: QUOTES_IN_DELAY,
-          useNativeDriver: true,
-        }),
-        Animated.timing(sizeAnim, {
-          toValue: 1,
-          duration: QUOTES_IN_DURATION,
-          delay: QUOTES_IN_DELAY,
-          useNativeDriver: true,
-        }),
-      ]),
-    ]).start();
-    const endOfAnim = setTimeout(() => {
-      setIsRegistered(false);
-      clearTimeout(endOfAnim);
-    }, Math.max(QUOTES_OUT_DURATION, SVG_IN_DURATION) + SVG_IN_DELAY + Math.max(SVG_OUT_DURATION, QUOTES_IN_DURATION) + QUOTES_IN_DELAY);
+  } else if (isShakeTriggered && hasInitialTransitionRun && !isQuoteAnimationRunning) {
+    setIsQuoteAnimationRunning(true);
+    quoteAnimationOut_CallbackIn();
   }
 
   return (
     <Animated.View style={{
-      ...props.style,
```

Comment:
No longer utilizing `props` object in component, nor passing styles through props.

Reply:
remove this comment